src/objects/conf/base.py

```diff
@@ -108,6 +111,8 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "axes.middleware.AxesMiddleware",
     "django_otp.middleware.OTPMiddleware",
+    "vng_api_common.middleware.AuthMiddleware",
+    "vng_api_common.middleware.APIVersionHeaderMiddleware",
```

Comment:
Why are they added here?

Reply:
In the documentation it says that I need to add these middleware:

[Documentation](https://vng-api-common.readthedocs.io/en/stable/quickstart.html)